fineract-provider/src/main/java/org/apache/fineract/cob/loan/CheckLoanRepaymentOverdueBusinessStep.java

```diff
@@ -71,4 +77,12 @@ public String getEnumStyledName() {
     public String getHumanReadableName() {
         return "Check loan repayment overdue";
     }
+
+    private static boolean isOverDueEventNeededToBeSent(Loan loan, Long numberOfDaysBeforeDueDateToRaiseEvent, LocalDate currentDate,
```

Comment:
Please do loan status and total outstanding check as the very first step of this business step!

Reply:
Done! removed the Loan status

Reply:
I dont see anywhere the loan status check... and the total outstanding check was also not moved.

I didnt say **remove** them, i said move those checks to be the very first conditions of this business step!

Reply:
Done!